.github/workflows/build-main.yml

```diff
@@ -0,0 +1,74 @@
+name: Build Main
+
+on:
+  push:
+    branches:
+      - 'main'
+  schedule:
+    - cron:  '12 0 * * *'
```

Comment:
~[curious-question]: Is it needed to build the image daily?~

Also, does this mean "Daily At 12:12 AM"?

Reply:
Auto answer: Yes, to bring the changes from Kaoto UI and Kaoto Backend that happen on other repositories.